packages/web/src/common/store/pages/saved/lineups/sagas.ts

```diff
@@ -60,9 +60,10 @@ function* getTracks({ offset, limit }: { offset: number; limit: number }) {
   }, {})
 
   const localLibraryAdditions = yield* select(getSelectedCategoryLocalTrackAdds)
-  const localLibraryAdditionsTrackIds = Object.keys(
-    localLibraryAdditions
-  ).filter((savedTrackId) => !savedTrackTimestamps[savedTrackId])
+  const localLibraryAdditionsTrackIds = Object.keys(localLibraryAdditions)
+    .filter((savedTrackId) => !savedTrackTimestamps[savedTrackId])
+    .map((trackId) => parseInt(trackId, 10))
```

Comment:
`Number(trackId)` prob simpler than `parseInt` but doesn't really matter if it works

Reply:
good call, will update